superset/common/query_object_factory.py

```diff
@@ -21,25 +21,25 @@
 
 from superset.common.chart_data import ChartDataResultType
 from superset.common.query_object import QueryObject
+from superset.dao.datasource import DatasourceDAO
```

Comment:
use `superset.DatasourceDAO` here like ConnectorRegistry was below in the type_checking